models/rarible/polygon/rarible_polygon_schema.yml

```diff
@@ -0,0 +1,246 @@
+version: 2
+
+models:
+  - name: rarible_polygon_events
+    meta:
+      blockchain: polygon
+      project: rarible
+      contributors: springzh
+    config:
+      tags: ['polygon','rarible','events']
+    description: >
+        Rarible events on Polygon
+    columns:
```

Comment:
We should include a uniqueness test here for the unique_key columns

Reply:
Added unique test to unique_trade_id field